backends/cloud/cloudmanager.cpp

```diff
@@ -124,7 +124,7 @@ void CloudManager::save() {
 		Common::String name = getStorageConfigName(i);
 		ConfMan.set(kStoragePrefix + name + "_username", _storages[i].username, ConfMan.kCloudDomain);
 		ConfMan.set(kStoragePrefix + name + "_lastSync", _storages[i].lastSyncDate, ConfMan.kCloudDomain);
-		ConfMan.set(kStoragePrefix + name + "_usedBytes", Common::String::format("%llu", _storages[i].usedBytes), ConfMan.kCloudDomain);
+		ConfMan.set(kStoragePrefix + name + "_usedBytes", Common::String::format("%llu", (unsigned long long)_storages[i].usedBytes), ConfMan.kCloudDomain);
```

Comment:
It is better use `uint64` here and in other places.

Reply:
But that's exactly the problem I tried to solve... `uint64` is defined as `unsigned long` on Linux64, so `%llu` doesn't match the type. And on Windows and Linux32 it is `unsigned long long`, so `%lu` doesn't match it.